.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
```

Comment:
Is this file related to this task??

Reply:
As per the comments, the `travis.yml` file was not related to the validations hence I removed it from the new branch I have created to house the validations (`ft-signup-validators-160577607` which houses the requested changes.